File: BeeCrowd/Python/beecrowd_1187.py
add, count = 0, 0

op = input()

for lin in range(12):
    for col in range(12):
        n = float(input())
        if (lin < col and col > lin and col < 11-lin):
            add += n
            count += 1

print('%.1f' %(add if op=='S' else add/count) )

File: BeeCrowd/Cpp/beecrowd_1187.cpp
#include <stdio.h>

int main() {
  int lin, col;
  char op;
  double n, add = 0, count = 0;

  scanf("%c", &op);

  for (lin = 0; lin < 12; ++lin) {
    for (col = 0; col < 12; ++col) {
      scanf("%lf", &n);
      if (lin < col && col > lin && col < 11 - lin) {
        add += n;
        count += 1;
      }
    }
  }

  printf("%.1lf\n", (op == 'S') ? add : add / count);

  return 0;
}
